### PR TITLE
LPS-142832 Add com.liferay.dynamic.data.mapping.model.DDMFormRule to the XStreamConfigurator to avoid the com.thoughtworks.xstream.security.ForbiddenClassException error

### PR DESCRIPTION
**Pull request by jorgediaz-lr**
Related Issue: https://issues.liferay.com/browse/LPS-142832

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java [hunk 1]
```diff
@@ -71,6 +72,7 @@ protected void activate() {
 				com.liferay.dynamic.data.mapping.model.LocalizedValue.class),
 			new XStreamType(
 				com.liferay.dynamic.data.mapping.storage.DDMFormValues.class),
+			new XStreamType(DDMFormRule.class),
 			new XStreamType(DDMFormField.class),
 			new XStreamType(DDMFormFieldOptions.class),
 			new XStreamType(DDMFormFieldValidation.class),
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java:78]: Hi @jorgediaz-lr. 
Just one question, is it possible to sort this alphabetically? `DDMFormRule` after `DDMFormFieldValidation`. Thanks!

Comment (jorgediaz-lr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java]: Hi @carolmariaabb 

I have updated the PR with the correct order see new commit https://github.com/liferay-forms/liferay-portal/pull/1155/commits/5e93fa8758a3a78ca2b284a5e92ae993edda57cb

**Comment by jorgediaz-lr** [comment]
ci:test:sf

**Comment by jorgediaz-lr** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java]: Thanks @jorgediaz-lr!

**Review by carolmariaabb — APPROVED** [review]

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by jorgediaz-lr** [comment]
The test failures are unrelated to my changes, as they where executed correctly here: https://github.com/liferay-forms/liferay-portal/pull/1155#issuecomment-974891339 and after that test we have only changes the code order

**Comment by liferay-forms** [comment]
Hey @jorgediaz-lr, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @jorgediaz-lr, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**